rpm/openssl-pkcs11-sign-provider.spec

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2023 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+Name:          openssl-pkcs11-sign-provider
+Version:       1.0.0
+Release:       %autorelease
+Summary:       A PKCS#11 provider for OpenSSL 3.0 (private key operations only)
+License:       Apache-2.0
+URL:           https://github.com/opencryptoki/%{name}
+Source:        %{url}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+
+Requires:      openssl >= 3.0.8
```

Comment:
Just to make sure: the provider is not s390x specific? Otherwise we would need a
ExclusiveArch:	s390x

Reply:
The provider is not s390-specific and can be used on all platforms. So `ExclusiveArch` is not required.